helium/src/scss/_course_header.scss

```diff
@@ -51,21 +51,54 @@
   }
 }
 
-.page_header {
+.course-page, .lesson-page, .module-page {
+  #module_nav {
+    @include respond_to(tablet_landscape) {
+      h1, h2 {
+        padding-left: $ASIDE_WIDTH + 36px;
+      }
+      .banner_link {
+        padding-left: $ASIDE_WIDTH+10px;
+        width: 1024px + $ASIDE_WIDTH+10px;
+        overflow: hidden;
+      }
+    }
+  }
+}
+
+#yc-page .page_header {
```

Comment:
adding #yc-page, is to cater for the increased specificity of the typography (under #yc-root in theme), otherwise these settings get ignored. the padding below is modified in response to this change to maintain the same visual style.